bundles/admin/javascript-hooks/index.rst

```diff
@@ -0,0 +1,12 @@
+Javascript-Hooks
+================
+
+Javascript-hooks are just special-name-properties placed in the return-object of
+javascript-components. The AdminBundle recognizes the hooks and does stuff, like rendering
```

Comment:
`does stuff` doesn't sound very professional, would use `execute functionality` or something like this.